tests/functional/olp-cpp-sdk-dataservice-read/CatalogClientTest.cpp

```diff
@@ -142,6 +212,74 @@ TEST_P(CatalogClientTest, GetCatalog) {
   EXPECT_SUCCESS(catalog_response);
 }
 
+TEST_F(CatalogClientTest, GetVersionsList) {
+  const auto catalog = olp::client::HRN::FromString(kTestHrn);
+  auto client = std::make_unique<olp::dataservice::read::CatalogClient>(
+      catalog, settings_);
+  {
+    SCOPED_TRACE("Get versions list online");
+    {
+      mock_server_client_->MockAuth();
+      mock_server_client_->MockTimestamp();
```

Comment:
I don't think these are necessary anymore. Timestamp should be disabled by default.